.spacelift/config.yml

```
@@ -1,7 +1,16 @@
 # The version of the configuration file format
 version: 1
 # Your module version - must be changed to release a new version
-module_version: 0.1.0
+module_version: 0.1.1
```

Since you are removing `var.force_delete`, this is a breaking change and the MAJOR version of the module should be incremented instead of the PATCH version (which is reserved for backward compatible bug fixes). 
```suggestion
module_version: 1.0.0
```

If you amend the PR to keep `var.force_delete` while still adding the ability to override the force_delete configuration on a per-repo basis, you'd still increment the MINOR version as follows:
```suggestion
module_version: 0.2.0
```